great_expectations/experimental/metric_repository/column_descriptive_metrics_metric_retriever.py

```diff
@@ -143,11 +143,11 @@ def _get_table_column_types(
             aborted_metrics=aborted_metrics,
         )
         raw_column_types: list[dict[str, Any]] = value
-        # If type is not found, default to UNKNOWN
+        # If type is not found, default to empty string. This can happen if our db introspection fails.
         column_types_converted_to_str: list[dict[str, str]] = [
             {
                 "name": raw_column_type["name"],
-                "type": str(raw_column_type.get("type", "UNKNOWN")),
+                "type": str(raw_column_type.get("type", "")),
```

Comment:
Could you leave the `type` field out?
Although my assumption is that is a required field.

Reply:
Currently it's required e.g. https://github.com/greatexpectationslabs/mercury/blob/develop/services/ge_cloud/mercury/handlers/gql/metric.py#L136. I could probably loosen up the endpoint though - I think @donaldheppner was asking about this as well. I'll merge this for now but take a crack first thing tomorrow.

Reply:
This PR is for the backend side: https://github.com/greatexpectationslabs/mercury/pull/2364, I'll need a separate PR for OSS.